[DEVX-1549] Add docs for saucectl artifacts folder (#1224)
saucelabs/sauce-docs: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

#### `directory`
<p><small>| OPTIONAL | STRING |</small></p>

> Specifies the path to the folder location in which to download artifacts. A separate subdirectory is generated in this location for each suite for which artifacts are downloaded. The name of the subdirectory is composed by the suite name. If there were subdirectories with the same name already existed, the new one would be appended a serial number.

Reviewer: ```suggestion
Specifies the path to the folder location in which to download artifacts. A separate subdirectory is generated in this location for each suite for which artifacts are downloaded. The name of the subdirectory will match the suite name. If a directory with the same name already exists, the new one will be suffixed by a serial number.
```
